docs/platforms/rust/common/logs/index.mdx

```diff
@@ -130,23 +119,25 @@ fn main() {
 
 The fields of `tracing` events are automatically captured as attributes in Sentry logs.
 
-To map a `tracing` event to multiple items in Sentry (e.g. both an event and a log), combine multiple event filters using the bitwise or operator.
+By default, `tracing` events at info level or above are captured as Sentry logs.
+
+This behavior can be customized by applying a custom `event_filter` when creating the layer.
+The following snippet shows the default `event_filter` that's applied when using the `sentry` crate with the `logs` feature flag.
 
 ```rust
 use sentry::integrations::tracing::EventFilter;
 
 let sentry_layer =
     sentry::integrations::tracing::layer().event_filter(|md| match *md.level() {
         // Capture error and warn level events as both logs and events in Sentry
-        tracing::Level::ERROR | tracing::Level::WARN => EventFilter::Event | EventFilter::Log,
+        tracing::Level::ERROR => EventFilter::Event | EventFilter::Log,
         // Ignore trace level events, as they're too verbose
         tracing::Level::TRACE => EventFilter::Ignore,
-        // Capture everything else just as a log
-        _ => EventFilter::Log,
+        // Capture everything else as both a breadcrumb and a log
+        _ => EventFilter::Breadcrumb | EventFilter::Log,
     });
-```
 
-### `log` Integration
+### `log` crate Integration
```

Comment:
Also why are we changing the name here?

l: I find it weird that we call this integration "`log` crate" but the other one just "`tokio-rs/tracing`" without the word "crate" – we should stay consistent.